django_cprofile_middleware/middleware.py

```diff
@@ -47,10 +53,20 @@ def process_view(self, request, callback, callback_args, callback_kwargs):
 
     def process_response(self, request, response):
         if self.can(request):
-            self.profiler.create_stats()
-            io = StringIO()
-            stats = pstats.Stats(self.profiler, stream=io)
-            stats.strip_dirs().sort_stats(request.GET.get('sort', 'time'))
-            stats.print_stats(int(request.GET.get('count', 100)))
-            response.content = '<pre>%s</pre>' % io.getvalue()
+            if 'download' in request.GET:
+                temp_file = tempfile.NamedTemporaryFile(suffix='.prof',
```

Comment:
Also, would it be possible to do this using a StringIO object instead of a `NamedTemporaryFile` ?